web/app/js/components/PodDetail.jsx

```diff
@@ -40,7 +40,7 @@ export default class PodDetail extends React.Component {
       pod: pod,
       upstreamMetrics: [],
       downstreamMetrics: [],
-      summaryMetrics: {},
+      podTs: {},
       pendingRequests: false,
       loaded: false
     };
```

Comment:
Should you also include `upstreamTsByPod` and `downstreamTsByPod` in the empty state here?